config.js

```diff
@@ -114,6 +111,25 @@ module.exports = {
     modelDirName: 'model',
     passwordHashSaltNumRounds: 8,
   },
+  redis: {
+    channelName: 'focus',
+    instanceUrl: {
+      cache:
+        pe.REDIS_CACHE || pe.REDIS_URL || DEFAULT_LOCAL_REDIS_URL,
+      limiter:
+        pe.REDIS_LIMITER || pe.REDIS_URL || DEFAULT_LOCAL_REDIS_URL,
+      pubsub:
+        pe.REDIS_PUBSUB || pe.REDIS_URL || DEFAULT_LOCAL_REDIS_URL,
+      queue:
+        pe.REDIS_QUEUE || pe.REDIS_URL || DEFAULT_LOCAL_REDIS_URL,
+      realtimeLogging:
+        pe.REDIS_REALTIME_LOGGING || pe.REDIS_URL || DEFAULT_LOCAL_REDIS_URL,
+      sampleStore:
+        pe.REDIS_SAMPLE_STORE || pe.REDIS_URL || DEFAULT_LOCAL_REDIS_URL,
+      session:
+        pe.REDIS_SESSION || pe.REDIS_URL || DEFAULT_LOCAL_REDIS_URL,
+    },
```

Comment:
We don't *need* to... but once I was identifying all the different uses, I just went with making *all* of them configurable, giving admin the ability to leave stuff on the main redis instance or move anything off to a separate instance.